default.nix

```diff
@@ -43,6 +43,15 @@ stdenv.mkDerivation rec {
     # CoqIDE dependencies
     ocamlPackages.lablgtk
 
+  ] else []) ++ (if buildDoc then [
+
+    # Sphinx doc dependencies
+    ocamlPackages.lablgtk
+    pkgconfig (python3.withPackages
```

Comment:
Could you put this on a new line for more clarity? Otherwise, it may look like a function application to the untrained eye.